### ISSUE TITLE
Handle connection errors in revision_score_importer

### ISSUE THREAD

**Issue by thenickcox** [WikiEducationFoundation/WikiEduDashboard#326]
Importing all unscored revisions in the batch hits what I'm assuming is a rate limit from ores.wmflabs.org. With my set of data, batches of 50 ended up getting separated into 656 batches. After 491, the connection dropped:

```sh
"Pulling revisions: batch 485 of 656"
"Pulling revisions: batch 486 of 656"
"Pulling revisions: batch 487 of 656"
"Pulling revisions: batch 488 of 656"
"Pulling revisions: batch 489 of 656"
"Pulling revisions: batch 490 of 656"
"Pulling revisions: batch 491 of 656"
rake aborted!
Errno::ECONNREFUSED: Connection refused - connect(2) for "ores.wmflabs.org" port 80
/Users/nickcox/projects/WikiEduDashboard/lib/importers/revision_score_importer.rb:89:in `get_revision_scores'
/Users/nickcox/projects/WikiEduDashboard/lib/importers/revision_score_importer.rb:15:in `block in update_revision_scores'
/Users/nickcox/projects/WikiEduDashboard/lib/importers/revision_score_importer.rb:12:in `each'
/Users/nickcox/projects/WikiEduDashboard/lib/importers/revision_score_importer.rb:12:in `each_slice'
/Users/nickcox/projects/WikiEduDashboard/lib/importers/revision_score_importer.rb:12:in `with_index'
/Users/nickcox/projects/WikiEduDashboard/lib/importers/revision_score_importer.rb:12:in `update_revision_scores'
/Users/nickcox/projects/WikiEduDashboard/lib/tasks/batch.rake:155:in `block (3 levels) in <top (required)>'
```

We should handle that, potentially by sleeping and retrying.

**Comment by thenickcox** [comment]
This apparently isn't due to rate limiting. Leave as is.

**Comment by ragesoss** [comment]
We should expect this error, though, and handle it without aborting the rake.

**Comment by ragesoss** [comment]
Fixed with ebd82a768dcf4779fa5a469930134540fc0d4496
